test.ini

```diff
@@ -0,0 +1,45 @@
+[DEFAULT]
```

Comment:
用途がわからなかったのですが、このファイルはどのように使われるのでしょうか？

Reply:
テスト時にckanのconfigファイルとしての動作をするようです。

```
The test.ini file is a CKAN config file just like your /etc/ckan/default/ckan.ini file, and it can contain any [CKAN config file settings](https://docs.ckan.org/en/2.9/maintaining/configuration.html) that you want CKAN to use when running your tests
```
https://docs.ckan.org/en/2.9/extensions/testing-extensions.html

Reply:
テスト用ファイルであればルートディレクトリには置きたくないですね。
`tests`以下とか、`tests/config`以下とか、`tests/misc`以下とかでしょうか。

Reply:
承知いたしました。

```test.ini```はconfigファイルとして動作するようですので、```tests/config```以下に格納いたしました。
また合わせて、```test.yml```内で```test.ini```に関連する記述を修正いたしました。

Reply:
>  また合わせて、test.yml内でtest.iniに関連する記述を修正いたしました。
`test.yml`の最終更新は2月6日のようですが、コミット漏れでしょうか？
コメントアウトされている行なので大きな問題はありませんが、`test.ini`へのパスは直しておきたいですね。

Reply:
大変失礼いたしました。
コミットが漏れておりました。

```test.yml```の```test.ini```へのパスを3箇所、書き換えを行いました。